rollup/circuitcapacitychecker/impl.go

```diff
@@ -17,15 +17,26 @@ import (
 	"github.com/scroll-tech/go-ethereum/log"
 )
 
-type CircuitCapacityChecker struct{}
+// global count for total CircuitCapacityChecker
+var cnt uint8
```

Comment:
Sorry, just find out if we could do this fix.
Should we add a new function as `circuit_capacity_checker_count` to return `CHECKERS.len()` and delete this global count in go side?

Reply:
good idea. I will fix it

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/374/commits/1284caf864d0cba52b7ae60234c97df0006cfaa5